openmc/capi/math.py

```diff
@@ -182,7 +217,57 @@ def calc_zn_rad(n, rho):
     zn_rad = np.zeros(num_bins, dtype=np.float64)
     _dll.calc_zn_rad_c(n, rho, zn_rad)
     return zn_rad
-    
+
+
+def evaluate_zernike_rad(data, r):
+    """ Finds the value of f(x) given a set of even order Zernike coefficients
+    and the value of r.
+
+    Parameters
+    ----------
+    data : iterable of float
+        Legendre coefficients
+    r : float
+        Independent variable to evaluate the radial part of Zernike at
+
+    Returns
+    -------
+    float
+        Corresponding Zernike expansion result
+
+    """
+
+    data_arr = np.array(data, dtype=np.float64)
+    n = int(2 * (len(data) - 1))
+    return _dll.evaluate_zernike_rad_c(n,
+                                        data_arr.ctypes.data_as(POINTER(
+                                        c_double)),
+                                        r)
+
+
+def calc_norm_zn_rad(n, rho):
+    """ Calculate normalization arrays for the even orders in modified Zernike
+    polynomial moment with no azimuthal dependency (m=0).
+
+    Parameters
+    ----------
+    n : int
+        Maximum order
+    rho : float
+        Radial location in the unit disk
+
+    Returns
+    -------
+    numpy.ndarray
+        Corresponding resulting list of normalization coefficients
+
+    """
+
+    num_bins = n // 2 + 1
+    norm_zn_rad = np.zeros(num_bins, dtype=np.float64)
+    _dll.calc_norm_zn_rad_c(n, rho, norm_zn_rad)
```

Comment:
Same comment as with the Legendre case -- this can be easily implemented in Python/numpy.